packages/clarity-js/src/data/metric.ts

```diff
@@ -5,7 +5,9 @@ import encode from "./encode";
 export let data: MetricData = null;
 export let updates: MetricData = null;
 let metricMap: WeakMap<Node, string> = null; // Maps metric nodes => innerText
-const regex = /[^0-9\.]/g;
```

Comment:
NAive JS question: Is it more efficient to put the consts here rahter than in the function that use them (where it's easier to read the code as they are next to each other)?

Reply:
It's a good question. It's possible things have improved over time and browsers are smart about handling it internally now but here's a link that provides some additional info: https://stackoverflow.com/questions/8814009/how-often-does-javascript-recompile-regex-literals-in-functions